types/evaporate/index.d.ts

```diff
@@ -1,12 +1,117 @@
-// Type definitions for EvaporateJS
+// Type definitions for EvaporateJS 2.1
 // Project: https://github.com/TTLabs/EvaporateJS
-// Definitions by: Andrew Kuklewicz <https://github.com/kookster>, Chris Rhoden <https://github.com/chrisrhoden>
+// Definitions by: Junyoung Clare Jang <https://github.com/ailrun>
```

Comment:
This should preserve the old authors and add yourself at the end.

Reply:
Oh, I thought I can restart from the scratch without refering them when I made a typing for new version....
Thank you for pointing this